WcaOnRails/app/jobs/application_job.rb

```diff
@@ -1,4 +1,19 @@
 # frozen_string_literal: true
 
 class ApplicationJob < ActiveJob::Base
+  class DeferJob < StandardError
+    attr_reader :wait_time
+    def initialize(wait_time)
+      super("Delay the job by #{time} s")
```

Comment:
Good thought, but since we *always* rescue from this exception, is this string going to be displayed anywhere?